content/shmem_malloc.tex

```diff
@@ -21,10 +21,10 @@
 
 
 \apidescription{
-    The \FUNC{shmem\_malloc} routine returns a pointer to a block of at least
-    \VAR{size} bytes suitably aligned for any use.  This space is allocated from the
-    symmetric heap (in contrast to \FUNC{malloc}, which allocates from the private
-    heap).
+    The \FUNC{shmem\_malloc} routine returns a pointer to a block, of at least
+    \VAR{size} bytes, suitably aligned for any use.  This space is allocated from the
```

Comment:
It is not correct to set off the clause "of at least _size_ bytes". This is essential to the behavioral definition of `shmem_malloc` and therefore violates [Comma Use Rule #4](https://owl.english.purdue.edu/owl/resource/607/02/).